nirum_wsgi.py

```diff
@@ -156,20 +173,15 @@ def __init__(self, service,
                 )
             rules.append((
                 uri_template,
-                path_pattern,
+                matcher,
                 http_verb,
                 method_name  # Service method
             ))
```

Comment:
As it's getting more complex, it would be better if we define a named tuple for this.